src/shaders.rs

```diff
@@ -10,7 +10,7 @@ void main()
 }
 ";
 
-pub static VS_COLORED_150_CORE: &'static str = "
+pub const VS_COLORED_150_CORE: &'static str = "
```

Comment:
To hold a reference we do not need to allocate a static memory. The str itself in any case in static memory.